src/PerfView/UserCommands.cs

```diff
@@ -1512,6 +1533,37 @@ private TraceEvents GetTraceEventsWithProcessFilter(ETLDataFile etlFile)
             return events;
         }
 
+        /// <summary>
+        /// Save the CPU stacks for an ETL file into a perfView.xml.zip file.
+        /// </summary>
+        /// <param name="etlFile">The ETL file to save.</param>
+        /// <param name="process">The process to save. If null, save all processes.</param>
+        /// <param name="filter">The filter to apply to the stacks. If null, apply no filter.</param>
+        /// <param name="outputName">The name of the file to output data to. If null, use the default.</param>
+        private static void SaveCPUStacksForProcessAsCsv(ETLDataFile etlFile, TraceProcess process = null, FilterParams filter = null, string outputName = null)
```

Comment:
the output name is always null, so you can probably just get rid of that parameter

Reply:
I guess it's

Reply:
same thing with filter, but I guess the SaveCPUStacksForProcess does the same thing below so you could keep both, get rid of both, or make SaveCPUStacks be able to take in a filter and outputName

Reply:
Fixed

Reply:
Just going to deprecate both since they arent used at the moment